monzo/monzo.py

```diff
@@ -57,15 +58,18 @@ def get_first_account(self):
             raise LookupError('There are no accounts associated with this user.')
         return accounts['accounts'][0]
 
-    def get_transactions(self, account_id):
+    def get_transactions(self, account_id, parse_datetimes=True):
         """Get all transactions of a given account. (https://monzo.com/docs/#list-transactions)
 
            :param account_id: The unique identifier for the account which the transactions belong to.
+           :param parse_datetimes: Enable parsing of timestamp strings to datetime objects.
            :rtype: A collection of transaction objects for specific user.
         """
         url = "{0}/transactions".format(self.API_URL)
         params = {'expand[]': 'merchant', 'account_id': account_id}
         response = self.request.get(url, headers=self.headers, params=params)
+        if parse_datetimes:
+            response = self.parse_datetimes(response)
```

Comment:
You haven't broken out of the first layer of the dictionary here.

Try `response = self.parse_datetimes(response['transactions'])`

Reply:
You might want to wrap it back up in a dictionary again after for consistency.

I'd quite like to unwrap the first level of returned dictionaries with a single element as standard but I'm waiting for my PR to merge first so it can be done consistently.

Reply:
It actually works fine in my own testing. The dictionary is broken out in line 253. ` for txn in transactions['transactions']:`

The problem appears to be that the mocked response used in the test suite does not invoke the `parse_datetimes` function.

Reply:
Fixed the test stub and it's now a green light from Travis :smile: